spec/shr_foundation.txt

```diff
@@ -354,3 +387,76 @@ Description:    "A set of codes drawn from one or more code systems"
+0..1		MasterIdentifier
+			StatusOrActive is type Status
+1..1		Status from http://hl7.org/fhir/ValueSet/document-reference-status
+0..1		DocumentStatus from http://hl7.org/fhir/ValueSet/composition-status
+1..1		Type from http://hl7.org/fhir/ValueSet/c80-doc-typecodes
+0..1		Category from http://hl7.org/fhir/ValueSet/c80-doc-classcodes
+0..1		SubjectOfRecord value is type ref(Patient)  // us core restriction 
+0..1		CreationDateTime
+1..1		StatementDateTime
+0..*		Author value is type PersonOrOrganizationOrDevice
+0..1		Authenticator
+0..1		ManagingOrganization
+1..*		DocumentReferenced
+0..*		RelatedDocument
+0..1		CommentOrDescription
+0..*		SecurityLabel from http://hl7.org/fhir/ValueSet/security-labels if covered
+0..1		ClinicalContext
+
+	Element:	MasterIdentifier
+	Description:	"Document identifier as assigned by the source of the document. This identifier is specific to this version of the document. This unique identifier may be used elsewhere to identify this version of the document."
+	Value:  Identifier
+
+	Element:	DocumentStatus
+	Description: "The status of the underlying document."
+	Value: CodeableConcept
+
+	Element:  Authenticator
+	Description:	"Who authenticated the document"
+	Value:	ref(Practitioner) or ref(Organization)
+
+	Element:	DocumentReferenced
+	Description:	"The document referenced."
+	1..1	Attachment
+	1..1	Attachment.ContentType  //  constraint from us-core
+	1..1	Attachment.ResourceLocation   //  constraint from us-core
+	0..1	Type from http://hl7.org/fhir/ValueSet/formatcodes if covered // us-core extensible binding
+
+	Element:  RelatedDocument
+	Description:	"Documents related to this one."
+	1..1	Relationship from http://hl7.org/fhir/ValueSet/document-relationship-type
+	1..1	ref(DocumentReference)
+
+		Element:  Relationship
+		Description:  "A description of the association between two items."
+		Value:	CodeableConcept
+	
+	Element:	ClinicalContext
+	Description:	"Clinical context of document"
+	0..1	ref(Encounter)  // in 4, changes to CareContext (Encounter or EpisodeOfCare)
+	0..*	Event
+	0..1	TimePeriod
+	0..1	FacilityType
+	0..1	Setting
+	0..1	ContemporaneousPatientInformation
+	0..*	RelatedInformation
+	0..*	Identifier
+
+		Element:	Event
+		Description:	"Main clinical acts documented"
+		Value:	CodeableConcept could be from http://hl7.org/fhir/ValueSet/v3-ActCode
+
+		Element:	FacilityType
+		Description:	"Kind of facility where patient was seen"
+		Value:	CodeableConcept could be from http://hl7.org/fhir/ValueSet/c80-facilitycodes
+
+		Element:	ContemporaneousPatientInformation
+		Description:	"he Patient Information as known when the document was published. May be a reference to a version specific, or contained."
```

Comment:
small typo.  Should read "The Patient Information" instead of "he Patient Information"